README.md

```diff
@@ -2,57 +2,258 @@
+
+### Dependencies
+
+In addition to the basic `vt-tv` dependencies listed above, you will also need:
+
+1. A Python version between 3.8 - 3.11
+2. [`nanobind`](https://nanobind.readthedocs.io/en/latest/), which can be installed with:
+
+```sh
+pip install nanobind
 ```
 
+### 1. Install
 
-Then, to install python-environment-wide the binded `vt-tv` python module, run:
+First, specify the location of your `VTK` build (see above) with:
 
 ```bash
-pip install <path/to/vt-tv/source>
+export VTK_DIR=/path/to/vtk/build
 ```
-**Optional**
 
-To specify the number of parallel jobs to use during the build, you can set the `VTTV_J` environment variable:
+Optional: To specify the number of parallel jobs to use during the build, you can set the `VT_TV_CMAKE_JOBS` environment variable:
 
 ```bash
-export VTTV_J=8
+export VT_TV_CMAKE_JOBS=8
 ```
 
-> [!NOTE]
-> Behind the scenes, the usual `cmake` and `make` commands are run. Depending on your system, this can cause the install process to be lengthy as it will be compiling the entire `vt-tv` library.
+Then install the binded `vt-tv` Python module with:
+
+```bash
+pip install ${VTTV_SOURCE_DIR}
+```
+
+_Note: Behind the scenes, the usual `cmake` and `make` commands are run. Depending on your system, this can cause the install process to be lengthy as it will be compiling the entire `vt-tv` library._
+
+---
+
+### 2. Usage
+
+Import the `vt-tv` module into your project using:
+
+```python
+import vttv
+```
+
+The only function you need is `vttv.tvFromJson`, which has the following (C++) function signature:
+
+```cpp
+void tvFromJson(
+    const std::vector<std::string>& input_json_per_rank_list,
+    const std::string& input_yaml_params_str,
+    uint64_t num_ranks
+)
+```
+
+The parameters are:
+- `input_json_per_rank_list`: A list of the input JSON data strings (one string per rank). In the C++ standalone app, this equates to the input JSON data files.
+- `input_yaml_params_str`: The visualization and output configuration data, formatted as a dictionary but exported as a string (see example below). This equates to the standalone app's input YAML configuration file.
+- `num_ranks`: The number of ranks to be visualized by `vt-tv`.
+
+As an example, here is the (emptied) code used by the [`Load Balancing Analysis Framework`](https://github.com/DARMA-tasking/LB-analysis-framework) to call `vt-tv`:
+
+```python
+import vttv
+
+# Populate with the JSON data from each rank
+ranks_json_str = []
+
+# Populate with the desired configuration parameters
+vttv_params = {
+    "x_ranks": ,
+    "y_ranks": ,
+    "z_ranks": ,
+    "object_jitter": ,
+    "rank_qoi": ,
+    "object_qoi": ,
+    "save_meshes": ,
+    "force_continuous_object_qoi": ,
+    "output_visualization_dir": ,
+    "output_visualization_file_stem":
+}
+
+# Populate with number of ranks used in the current problem
+num_ranks =
+
+# Call vt-tv
+vttv.tvFromJson(ranks_json_str, str(vttv_params), num_ranks)
+```
+</details>
+
+## Design Information
```

Comment:
@pierrepebay Are there any other design elements that we should discuss in the README?

Reply:
@cwschilly let's make sure this is addressed before merging (in case there is something else @pierrepebay wants to add before this branch gets merged)